packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -3228,5 +3228,28 @@ void main() {
         ),
       );
     });
+
+    testWidgets(
```

Comment:
This test is passing without the patch. Is the original issue already resolved?

Reply:
After looking into this, the "single tap hides" test was passing because of strangeness with the tap location.  Using a direct `longPress(textFieldFinder)` made the test fail on master and pass in this branch.

For the other test, "long press on autofocused", I could not get it to fail on master no matter what I tried.  However, trying a sample app, the behavior looked as if the test should definitely fail.  I gave up trying to make it fail and instead wrote an integration test that fails on master and passes on this branch.

Reply:
I think i know what is going on.
When you use longPressAt(textOffsetToPosition(tester, 0)), it will tap at the start of the text which is also part of the selection handle. this will trigger _handleSelectionHandleTapped which toggle the toolbar off. That is why it still passing in master.

As for long press on autofocused, add an additional pump after the pump widget will fail the test as the on focus callback will update the selection from the default (-1, -1) to (0,0) but it take one frame to propagate down to renderEditable. While the long press started, the renderEditable is still has selection(-1,-1) which allow it to update its selection

Reply:
Brilliant, that did in fact make it fail on master.  Thanks so much.  I will make that change and then remove the e2e test (since it's much slower).